app/models/attachment_data.rb

```diff
@@ -175,6 +175,10 @@ def auth_bypass_ids
     attachable && attachable.respond_to?(:auth_bypass_id) ? [attachable.auth_bypass_id] : []
   end
 
+  def access_limited_organisation_ids
+    organisations.pluck(:id).compact if access_limited?
+  end
```

Comment:
I can't find where this method is being used – is it still needed? I think it might be unused.